edx_exams/apps/lti/views.py

```diff
@@ -328,7 +329,7 @@ def end_assessment(request, attempt_id):
 
         return redirect(preflight_url)
 
-    return JsonResponse({})
+    return redirect(get_exam_url_path(exam.course_id, exam.content_id))
```

Comment:
this will leave us on a blank page with '{}'. If we don't need to perform an LtiEndAssessment launch return to the exam section.